templates/blocks/take_action_boxout.twig

```diff
@@ -1,63 +1,67 @@
 {% block take_action_boxout %}
 
-	{% if ( boxout ) %}
-		{% if ( boxout.image ) %}
-			{% set bg_image = 'style="background-image: linear-gradient(180deg, rgba(0, 0, 0, 0.3), rgba(0, 0, 0, 0)), url(\''~boxout.image~'\');"' %}
-		{% else %}
-			{% set bg_image = '' %}
-		{% endif %}
-		<section id="action-card" class="cover-card single-cover dark-card-bg action-card" {{ bg_image|raw }}>
-            <a
-                data-ga-category="Take Action Boxout"
-                data-ga-action="Image"
-                data-ga-label="n/a"
-                class="cover-card-overlay"
-                href="{{ boxout.link|default('#') }}"
-                {{ boxout.new_tab and boxout.link ? 'target="_blank"' }}
-            />
-            <div class="cover-card-content">
-                {% if ( boxout.campaigns ) %}
-                    {% for campaign in boxout.campaigns %}
-                        <a
-                            class="cover-card-tag"
-                            data-ga-category="Take Action Boxout"
-                            data-ga-action="Navigation Tag"
-                            data-ga-label="n/a"
-                            href="{{ campaign.link|e('esc_url') }}"
-                        >
-                            <span aria-label="hashtag">#<span>{{ campaign.name|e('wp_kses_post')|raw }}
-                        </a>
-                    {% endfor %}
-                {% endif %}
-                {% if ( boxout.title ) %}
-                    <a
-                        class="cover-card-heading"
-                        data-ga-category="Take Action Boxout"
-                        data-ga-action="Title"
-                        data-ga-label="n/a"
-                        href="{{ boxout.link|default('#') }}"
-                        {{ boxout.new_tab and boxout.link ? 'target="_blank"' }}
-                    >
-                        {{ boxout.title|e('wp_kses_post')|raw }}
-                    </a>
-                {% endif %}
-            </div>
-			{% if ( boxout.excerpt ) %}
-				<p>{{ boxout.excerpt|e('wp_kses_post')|excerpt(25)|raw }}</p>
+{% if ( boxout ) %}
+  {% if ( boxout.image ) %}
+    {% set bg_image = 'style="background-image: linear-gradient(180deg, rgba(0, 0, 0, 0.3), rgba(0, 0, 0, 0)), url(\''~boxout.image~'\');"' %}
```

Comment:
I think you have mixed indentation here. Both spaces and tabs. Twig files are using tabs.
If you are on VScode there is a "Convert indentation to tabs" option command